src/logging_unittest.cc

```diff
@@ -1586,3 +1586,12 @@ TEST(Logging, FatalThrow) {
   ScopedExit<decltype(restore_fail)> restore{restore_fail};
   EXPECT_THROW({ LOG(FATAL) << "must throw to fail"; }, std::logic_error);
 }
+
+TEST(DeathLogging, ErrorMessage) {
+  ASSERT_DEATH({ LOG(FATAL) << "foo"; }, "Check failure stack trace");
+  ASSERT_DEATH(
+      {
+        LOG_AT_LEVEL(google; : LogSeverity::FATAL) << "foo";
```

Comment:
This is not a valid C++ syntax which is probably the reason for the failing CI. I believe you wanted something like this:
```suggestion
        LOG_AT_LEVEL(google::LogSeverity::FATAL) << "foo";
```

Reply:
Whops, done.  I was having trouble running tests locally with bazel.